editor/plugins/script_editor_plugin.cpp

```diff
@@ -2556,6 +2556,10 @@ void ScriptEditor::save_all_scripts() {
 	_update_script_names();
 }
 
+bool ScriptEditor::is_floating() const {
+	return !make_floating->is_visible();
```

Comment:
That seems a bit hacky to decide if we're floating based on the visibility of a button.
I'd suggest adding a bool instead and setting it when making floating.